showcase/app/controllers/utilities.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { scheduleOnce } from '@ember/runloop';
+
+import type Owner from '@ember/owner';
+import type { Registry as Services } from '@ember/service';
+
+export default class UtilitiesController extends Controller {
+  @service router!: Services['router'];
```

Comment:
I wonder where this pattern came from?
```suggestion
@service declare readonly router: RouterService;
```

Reply:
Looks like it came from SideNav/AppSideNav
https://github.com/hashicorp/design-system/blob/f36277115f578ff14e0c4ae7f442f00b063caf00/packages/components/src/components/hds/app-side-nav/portal/target.ts#L23

https://github.com/hashicorp/design-system/blob/f36277115f578ff14e0c4ae7f442f00b063caf00/packages/components/src/components/hds/side-nav/portal/target.ts#L24